source/common/http/http1/codec_impl.h

```diff
@@ -39,6 +39,14 @@ namespace Http1 {
  * Wrapper struct for the HTTP/1 codec stats. @see stats_macros.h
  */
 struct CodecStats {
+  using AtomicPtr = Thread::AtomicPtr<CodecStats, Thread::AtomicPtrAllocMode::DeleteOnDestruct>;
```

Comment:
Would it make sense if I moved these structs to a common codec_stats header? I want to share these structs between codecs and their forked legacy versions to avoid duplicating `ClusterInfoImpl::http1CodecStats()`

Reply:
Would be fine with me.

Note that the http3/quic ones may be a bit different (@danzh2010 ) because there are more of them and the master-list of them is maintained in an external codebase.